source/MaterialXGraphEditor/Graph.cpp

```diff
@@ -1834,7 +1834,7 @@ void Graph::copyInputs()
                             else if (upNode->getInput())
                             {
 
-                                copyNode->inputPins[count]->_input->setInterfaceName(upNode->getName());
+                                copyNode->inputPins[count]->_input->setInterfaceInput(upNode->getName());
```

Comment:
Every call is now "save" as it will remove the `value` attribute if a non-empty name is set, otherwise it will remove the `interfacename`. A `interfacename` with an empty string could also be set which could create an invalid doc.